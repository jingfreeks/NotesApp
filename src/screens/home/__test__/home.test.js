import React from 'react';
import { renderWithProviders } from "../../../utils/testframework";
import HomeScreen from '../home';


describe('Home Screen', () => {
  it('Should work as expected to get snapshot', () => {
    const all = renderWithProviders(<HomeScreen />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
