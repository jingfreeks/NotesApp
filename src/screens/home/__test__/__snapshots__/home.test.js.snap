// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen Should work as expected to get snapshot 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "alignSelf": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <View
    alignType="center"
    justify="flex-start"
    style={
      [
        {
          "alignItems": "center",
          "flexBasis": 0,
          "flexDirection": "column",
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "flex-start",
        },
      ]
    }
  >
    <Image
      resizeMode="contain"
      size={150}
      source={
        {
          "testUri": "../../../src/assets/suggestempty.png",
        }
      }
      style={
        [
          {
            "height": 294.2647058823529,
            "width": 294.2647058823529,
          },
        ]
      }
    />
    <Text
      style={
        [
          {
            "color": "#000000",
            "fontFamily": "Avenir",
            "fontSize": 16,
            "fontWeight": "400",
          },
        ]
      }
    >
      No notes data found
    </Text>
  </View>
</View>
`;
